CSVSource/README.md

```diff
@@ -16,6 +16,8 @@ In this way, in cases where the extension is not running during file updates, it
 In addition to handling delimiter-separated fields, this connector can extract fields from fixed positions in the record.
 This is configured as described below.
 
+the extension source support docker, when running in docker is detected, the windows file path is changed to UNIX file path, so the same configuration can be used for docker and non-docker activate.
```

Comment:
when the system runs on a windows machine, no conversion is done to the file system, 
it is Vantiq recommendation to use docker with a Linux ( and not Microsoft ) as such when it runs in a container, it will always behave like Linux